### DISCUSSION TITLE
Local file picker support

### DISCUSSION THREAD

**Discussion by ezhilkarthi** [eclipse-theia/theia#9875] [General]
Hi,
I have one question. I am trying to implement a file picker with a selected extension, does thiea provide any support regarding this?

I'm trying to "tsx" format in this dropdown. 
Thanks.
![image](https://user-images.githubusercontent.com/31158252/128882175-d0c666c3-a79f-4851-80c8-1fa033697be4.png)

**Comment by vince-fugnitto** [comment]
@ezhilkarthi the native or local file picker is only accessible in the `electron` target of the framework, not the `browser` since the filesystem is not local. Were you aiming on having this picker in the browser?

**Reply by ezhilkarthi** [reply]
Yes, I'm aiming to have this picker in the browser kindly suggest a possible way.

**Reply by vince-fugnitto** [reply]
> Yes, I'm aiming to have this picker in the browser kindly suggest a possible way.

I don't think it is possible in the `browser`, if you were to trigger the native file-picker it would be where your browser is located, and not your possible remote filesystem.
